packages/cli/templates/default-backend-plugin/src/plugin.ts.hbs

```diff
@@ -26,6 +26,10 @@ export const {{pluginVar}} = createBackendPlugin({
             logger,
           }),
         );
+        httpRouter.addAuthPolicy({
+          path: '/health',
+          allow: 'unauthenticated',
+        });
```

Comment:
In a future world where Backstage instances are publicly deployed, could this allow for enumeration searches/exploits? ie seeing if a certain plugin is installed?

Reply:
What do you propose as a solution?